### DISCUSSION TITLE
Copyright

### DISCUSSION THREAD

**Discussion by GurjeetSahu** [ravynsoft/ravynos#189] [Q&A] [answered]
What if Apple imposes copyright infringement on you guys?

**Comment by mszoek** [comment, marked as answer]
For what? We don't use any Apple trademarks or proprietary code. Everything used is publicly available open-source licensed code or original code written by the project team. The right to use what code we do have from Apple is granted by them placing it under the Apache, APSL or BSD-style licenses.

**Reply by GurjeetSahu** [reply]
Thanks, installing it right now.
